runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,144 @@
 
 /* @ddr_namespace: map_to_type=J9JavaAccessFlags */
 
-/* Constants from J9JavaAccessFlags */
+/*
+ * Modifiers of ROM classes, fields or methods.
+ * When a given modifier is defined by the JVM specification,
+ * the value is taken from that specification.
+ */
+#define J9AccPublic 0x1
+#define J9AccPrivate 0x2
+#define J9AccProtected 0x4
+#define J9AccStatic 0x8
+#define J9AccFinal 0x10
```

Comment:
Rather than reorganizing these constants into groups based on what they apply to, perhaps we could add a comment to each listing the contexts in which it is relevant. Then it will be clearer which bits are in use for which purposes. Some examples:
```c
#define J9AccPublic 0x1 /* class field  method */
#define J9AccSynchronized 0x20 /* method */
#define J9AccValueType 0x40 /* class */
#define J9AccVolatile 0x40 /* field */
```
Then we would have groups for `modifier`, `classDepthAndFlags` and `extraModifiers`. Entries would be ordered by value within each group so if a bit is reused for different elements the comments make it clear that such reuse is not problematic (e.g. `J9AccValueType` and `J9AccVolatile` above).

Reply:
Great idea.

Reply:
I pushed a new commit with this change, let me know what you think.